Sources/GRPC/GRPCClientStateMachine.swift

```diff
@@ -544,21 +544,20 @@ extension GRPCClientStateMachine.State {
     // responses as well as a variety of non-GRPC content-types and to omit Status & Status-Message.
     // Implementations must synthesize a Status & Status-Message to propagate to the application
     // layer when this occurs."
-    let statusHeader = headers[":status"].first
-    let responseStatus = statusHeader.flatMap(Int.init).map { code in
+    let responseStatus = headers.first(name: ":status").flatMap(Int.init).map { code in
       HTTPResponseStatus(statusCode: code)
```

Comment:
`.map(HTTPResponseStatus.init)`?

Reply:
Unfortunately the init has a default second arg: `init(statusCode: Int, reasonPhrase: String = "")` so this isn't possible.